src/editor/EditorManager.js

```diff
@@ -778,6 +778,28 @@ define(function (require, exports, module) {
             handleFileRemoved(removedFiles);
         }
     }
+    function findEditProvider(){
+        var providers = _inlineEditProviders,
+            inlinePromise,// incase we want to send this back to thimble
+            i,
+            editor = getCurrentFullEditor(),
+            pos = editor.getCursorPos(),
+            providerRet;// incase we want to send this back to thimble
+
+
+        for (i = 0; i < providers.length && !inlinePromise; i++) {
+            var provider = providers[i].provider;
+            providerRet = provider(editor, pos);
+            if (providerRet) {
```

Comment:
ahh yes, that was my mistake, I was trying to fix the conflict and did not notice this.